pkgs/os-specific/linux/bcc/default.nix

```diff
@@ -19,7 +19,7 @@ python.pkgs.buildPythonApplication rec {
   buildInputs = [
     llvmPackages.llvm llvmPackages.clang-unwrapped kernel
     elfutils luajit netperf iperf
-    systemtap.stapBuild
+    systemtap.stapBuild flex
```

Comment:
`natveBuildInputs`?

Reply:
No, it *was* in there already.  But apparently it needs flex headers; I don't really know why.

Reply:
Ah. In that case it would need to be in both `buildInputs` and `nativeBuildInputs`.

Reply:
May also need `libfl.so`, FWIW, I don't remember the details for it but sometimes that's needed (or `libfl.a` or whatever).